src/pages/projectSettings/Header.tsx

```diff
@@ -11,6 +11,10 @@ import { HeaderButtons } from "./HeaderButtons";
 import { readOnlyTabs, WritableTabRoutes } from "./tabs/types";
 import { ProjectType } from "./tabs/utils";
 
+const notAvailableAtRepoLevel: Set<ProjectSettingsTabRoutes> = new Set([
```

Comment:
We have a similar utility for read-only tabs in `projectSettings/tabs/types.ts`; perhaps this should be located there too. Also maybe `projectOnlyTabs` is more clear? Your call!